azure/cognitive-services/main.tf

```diff
@@ -0,0 +1,19 @@
+resource "azurerm_resource_group" "main" {
+  name     = var.name
+  location = var.location
+  tags     = var.tags
+}
+
+resource "azurerm_cognitive_account" "main" {
+  name                  = var.name
+  location              = azurerm_resource_group.main.location
+  resource_group_name   = azurerm_resource_group.main.name
+  kind                  = var.kind
+  sku_name              = var.sku_name
+  custom_subdomain_name = var.name
+  tags                  = var.tags
+
+  identity {
```

Comment:
This identity block is redundant. Without it, the service gets a SystemAssigned identity. Since that's exactly what it's doing, we don't need it and the code can be a little shorter. 💪

Reply:
How does it get an identity? Just for being a terraform module?

Reply:
Every Azure _resource_ gets an identity. :)  Default is SystemAssigned, we can also have UserAssigned.